lib/ash_admin/components/resource/form.ex

```diff
@@ -1155,6 +1155,8 @@ defmodule AshAdmin.Components.Resource.Form do
                 name: @name || @form.name <> "[#{@attribute.name}]"
             }
           )}
+        <% Ash.Type.embedded_type?(@attribute.type) && match?(%{source: %AshPhoenix.FilterForm.Arguments{}}, @form) -> %>
```

Comment:
Does it not support embedded resources? Was this raising an error of some kind?

Reply:
the error I got is` (RuntimeError) Invalid inputs_for name nested_embed. Only :arguments is supported`, it is triggered https://github.com/ash-project/ash_phoenix/blob/v2.1.9/lib/ash_phoenix/filter_form/filter_form.ex#L1247)

Reply:
```
10:56:46.834 [error] #PID<0.707.0> running DemoWeb.Endpoint (connection #PID<0.701.0>, stream id 2) terminated
Server: localhost:4000 (http)
Request: GET /?domain=Accounts&resource=User&action_type=read&table=&primary_key=147e2490-ffcb-4e5c-ae52-439131016ac1
** (exit) an exception was raised:
    ** (RuntimeError) Invalid inputs_for name nested_embed. Only :arguments is supported
        (ash_phoenix 2.1.9) lib/ash_phoenix/filter_form/arguments.ex:126: Phoenix.HTML.FormData.AshPhoenix.FilterForm.Arguments.to_form/4
        (phoenix_live_view 1.0.0) lib/phoenix_component.ex:2649: Phoenix.Component."inputs_for (overridable 1)"/1
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/tag_engine.ex:96: Phoenix.LiveView.TagEngine.component/3
        (ash_admin 0.12.2) lib/ash_admin/components/resource/form.ex:1159: anonymous fn/2 in AshAdmin.Components.Resource.Form.render_attribute_input/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:414: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:555: anonymous fn/4 in Phoenix.LiveView.Diff.traverse_dynamic/7
        (elixir 1.16.1) lib/enum.ex:2528: Enum."-reduce/3-lists^foldl/2-0-"/3
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:412: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:582: anonymous fn/3 in Phoenix.LiveView.Diff.traverse_comprehension/5
        (elixir 1.16.1) lib/enum.ex:1826: Enum."-map_reduce/3-lists^mapfoldl/2-0-"/3
        (elixir 1.16.1) lib/enum.ex:1826: Enum."-map_reduce/3-lists^mapfoldl/2-0-"/3
        (elixir 1.16.1) lib/enum.ex:1826: Enum."-map_reduce/3-lists^mapfoldl/2-0-"/3
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:493: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:555: anonymous fn/4 in Phoenix.LiveView.Diff.traverse_dynamic/7
        (elixir 1.16.1) lib/enum.ex:2528: Enum."-reduce/3-lists^foldl/2-0-"/3
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:412: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:555: anonymous fn/4 in Phoenix.LiveView.Diff.traverse_dynamic/7
        (elixir 1.16.1) lib/enum.ex:2528: Enum."-reduce/3-lists^foldl/2-0-"/3
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:412: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:555: anonymous fn/4 in Phoenix.LiveView.Diff.traverse_dynamic/7
        (elixir 1.16.1) lib/enum.ex:2528: Enum."-reduce/3-lists^foldl/2-0-"/3
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:412: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:555: anonymous fn/4 in Phoenix.LiveView.Diff.traverse_dynamic/7
        (elixir 1.16.1) lib/enum.ex:2528: Enum."-reduce/3-lists^foldl/2-0-"/3
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:412: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:582: anonymous fn/3 in Phoenix.LiveView.Diff.traverse_comprehension/5
        (elixir 1.16.1) lib/enum.ex:1826: Enum."-map_reduce/3-lists^mapfoldl/2-0-"/3
        (elixir 1.16.1) lib/enum.ex:1826: Enum."-map_reduce/3-lists^mapfoldl/2-0-"/3
        (elixir 1.16.1) lib/enum.ex:1826: Enum."-map_reduce/3-lists^mapfoldl/2-0-"/3
        (elixir 1.16.1) lib/enum.ex:1826: Enum."-map_reduce/3-lists^mapfoldl/2-0-"/3
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:503: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:555: anonymous fn/4 in Phoenix.LiveView.Diff.traverse_dynamic/7
        (elixir 1.16.1) lib/enum.ex:2528: Enum."-reduce/3-lists^foldl/2-0-"/3
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:412: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:555: anonymous fn/4 in Phoenix.LiveView.Diff.traverse_dynamic/7
        (elixir 1.16.1) lib/enum.ex:2528: Enum."-reduce/3-lists^foldl/2-0-"/3
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:412: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:782: Phoenix.LiveView.Diff.render_component/8
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:720: Phoenix.LiveView.Diff.zip_components/5
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:703: anonymous fn/4 in Phoenix.LiveView.Diff.render_pending_components/6
        (stdlib 5.0.2) maps.erl:416: :maps.fold_1/4
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:648: Phoenix.LiveView.Diff.render_pending_components/6
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/diff.ex:143: Phoenix.LiveView.Diff.render/3
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/static.ex:288: Phoenix.LiveView.Static.to_rendered_content_tag/4
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/static.ex:171: Phoenix.LiveView.Static.do_render/4
        (phoenix_live_view 1.0.0) lib/phoenix_live_view/controller.ex:39: Phoenix.LiveView.Controller.live_render/3
        (phoenix 1.7.18) lib/phoenix/router.ex:484: Phoenix.Router.__call__/5
        dev.exs:29: DemoWeb.Endpoint.plug_builder_call/2
        deps/plug/lib/plug/debugger.ex:136: DemoWeb.Endpoint."call (overridable 3)"/2
        dev.exs:29: DemoWeb.Endpoint.call/2
        (plug_cowboy 2.7.2) lib/plug/cowboy/handler.ex:11: Plug.Cowboy.Handler.init/2
        (cowboy 2.12.0) /Users/esdras/Projects/ash_admin/deps/cowboy/src/cowboy_handler.erl:37: :cowboy_handler.execute/2
        (cowboy 2.12.0) /Users/esdras/Projects/ash_admin/deps/cowboy/src/cowboy_stream_h.erl:306: :cowboy_stream_h.execute/3
        (cowboy 2.12.0) /Users/esdras/Projects/ash_admin/deps/cowboy/src/cowboy_stream_h.erl:295: :cowboy_stream_h.request_process/3
        (stdlib 5.0.2) proc_lib.erl:241: :proc_lib.init_p_do_apply/3
```